website/versions.json

```diff
@@ -0,0 +1,4 @@
+[
+  "master",
```

Comment:
does this need to be master?  To me this implies nightly which we don't do.  Instead can we call it `latest` or something similar?  I'm open to other names.